lib/honeykiq/server_middleware.rb

```diff
@@ -47,9 +55,12 @@ def queue_fields(queue)
       }
     end
 
-    def duration_ms(event)
+    def run_event(event, msg, queue_name)
+      event.add(**default_fields(msg, queue_name))
+      event.add(**before_fields)
       start_time = Time.now
```

Comment:
Not related to this PR - but we should update to use thread clock not wall time for this, as we only care about the relative time here!